src/util/dnd.cpp

```diff
@@ -162,10 +159,10 @@ bool DragAndDropHelper::allowDeckCloneAttempt(
     }
 
     if (!event.mimeData()->hasText() ||
-                // prevent cloning to ourself
-                event.mimeData()->text() == group ||
-                // only allow clone from decks
-                !PlayerManager::isDeckGroup(event.mimeData()->text(), nullptr)) {
+        // prevent cloning to ourself
+        event.mimeData()->text() == group ||
+        // only allow clone from decks
+        !PlayerManager::isDeckGroup(event.mimeData()->text(), nullptr)) {
         return false;
     }
```

Comment:
Here is an other odd case. I think clang tries to match the opening brace of the if statement.
This case works well with original Google style where norm as indent is 2 and the if ( produces the desired double indent.

Reply:
ContinuationIndentWidth is set to 8. I have no idea why this is formatted in a wrong way.

Reply:
This works as expected: AlignAfterOpenBracket: DontAlign